Plugins/NativeEngine/Source/NativeEngine.cpp

```diff
@@ -616,7 +616,7 @@ namespace Babylon
                 const spirv_cross::ShaderResources resources = compiler.get_shader_resources();
                 assert(resources.uniform_buffers.size() == 1);
                 const spirv_cross::Resource& uniformBuffer = resources.uniform_buffers[0];
-#if (BGFX_CONFIG_RENDERER_METAL)
+#if __APPLE__
```

Comment:
Why this change? The comment below indicates that the guarded behavior is specific to Metal, not Apple; is that incorrect?

Reply:
BGFX_CONFIG_RENDERER_METAL not defined. __APPLE__ is the only define that I've found to make the guard.